### ISSUE TITLE
Run server no retry doesn't work

### ISSUE THREAD

**Issue by postelrich** [blaze/blaze#1313]
I tried running:

`Server(resources).run(retry=True)`

and got the following error.  The retry kwargs does not seem to be an allowed kwargs in the first try to start the server.  Probably need to pop it first. 

File "/Users/postelrich/anaconda/envs/dpx/lib/python3.4/site-packages/blaze/server/server.py", line 216, in run
    self.app.run(*args, port=port, **kwargs)
  File "/Users/postelrich/anaconda/envs/dpx/lib/python3.4/site-packages/flask/app.py", line 772, in run
    run_simple(host, port, self, **options)
TypeError: run_simple() got an unexpected keyword argument 'retry'

**Comment by llllllllll** [comment]
I have a fix for this, just cleaning up some stuff around it. Thanks for reporting this.

**Comment by cpcloud** [comment]
I thought we decided to can the retry behavior altogether

**Comment by llllllllll** [comment]
oh yeah. Maybe we could support it but make retry default to `False`? This way someone needed to explicitly mark that they want this to happen

**Comment by cpcloud** [comment]
@postelrich is this behavior critical for your application?

**Comment by postelrich** [comment]
No, its not critical but I see it could be a useful thing if there is a use
case for spinning up multiple servers. Though then they could always use
something like gunicorn for that.

You would just need to pop retry before passing it into run_simple.
